### DISCUSSION TITLE
INVALID USERNAME OR PASSWORD

### DISCUSSION THREAD

**Discussion by sodiq03** [wazuh/wazuh#19927] [Q&A] [answered]
Hi #community
I create a new user name on my wazuh cloud server but it didn't work. I have created multiple users but still the same. The error message whenever i tried to login with the new user credentials is INVALID USERNAME OR PASSWORD. TRY AGAIN. Anyway to go about this?

**Comment by KJTamas** [comment]
Hi,

Me too... 🤷‍♂️

**Reply by BoringKraken** [reply]
Did any one figure this out im not on a cloud server but im getting this error?

**Reply by sakib789** [reply, marked as answer]
Hi @BoringKraken 

Follow these documents to create a user in the Wazuh dashboard.
[Creating and setting a Wazuh admin user](https://documentation.wazuh.com/current/user-manual/user-administration/rbac.html#creating-and-setting-a-wazuh-admin-user)
[Creating and setting a Wazuh read-only user](https://documentation.wazuh.com/current/user-manual/user-administration/rbac.html#creating-and-setting-a-wazuh-read-only-user)

A Wazuh dashboard user creation has 3 main steps.

Creating an internal user from `Indexer management`  and assigning roles and permissions to the created user.

Next, mapping it to Wazuh from Server management.

And the final step is to ensure that the `run_as` value is set to `true` in the `/usr/share/wazuh-dashboard/data/wazuh/config/wazuh.yml `configuration file and restart the Wazuh dashboard service.

Check the documents; they have step-by-step guidelines.

Let me know if you need any further help on this.

You can also join any of our community platforms and get faster help related to Wazuh.
https://wazuh.com/community/
